_includes/navigation.html

```diff
@@ -1,23 +1,17 @@
 <header id="header" class="container-flex">
-  <!-- Logo -->
-  <img src="{{site.path}}/images/signalk.svg" alt="Signal K Logo"  height="100" width="100" />
-
-  <!-- Page Title -->
-  <h1><a href="{{site.path}}/" id="logo">Signal K</a></h1>
-
   <!-- Nav -->
-  <a class="{{page.index}}" href="{{site.path}}/index.html">Home</a>
-  <a class="{{page.overview}}" href="{{site.path}}/overview.html">Overview</a>
-  <a class="{{page.installation}}" href="{{site.path}}/installation.html">Installation</a>
-  <a class="{{page.documentation}}" href="{{site.path}}/specification/latest/">Documentation</a>
-  <a class="{{page.demo}}" href="{{site.path}}/demo.html">Application Demo</a>
-  <a class="{{page.team}}" href="{{site.path}}/team.html">Team</a>
+  <a class="{{page.index}}" href="{{site.path}}/index.html">HOME</a>
+  <a class="{{page.overview}}" href="{{site.path}}/overview.html">INFO</a>
+  <a class="{{page.installation}}" href="{{site.path}}/installation.html">INSTALL</a>
+  <a class="{{page.specification}}" href="{{site.path}}/specification.html">SPECS</a>
+  <a class="{{page.demo}}" href="{{site.path}}/demo.html">DEMO</a>
+  <a class="{{page.about}}" href="{{site.path}}/about.html">ABOUT</a>
```

Comment:
Make sure that the front matter variable in about.html is renamed from `team` to `about` to ensure correct highlighting on the nav bar.

Also, W.R.T. renaming pages, to be a good webizen, the old pages should be replaced with simple pages notifying users that the content has been moved. These should have a `redirect` variable in the front matter pointing to the new page. They can have an empty body or some explanatory text.